functions/go/set-standard-labels/transformer/set-standard-labels.go

```diff
@@ -0,0 +1,85 @@
+package transformer
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/set-labels/setlabels"
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var _ fn.Runner = &SetStandardLabels{}
+
+type SetStandardLabels struct {
+	forDeployment string
+	// TBD: validate whether the recommended labels from the ResourceList.Items are set as expected.
+	validateOnly bool
```

Comment:
Would be nice to have a separate function (or just have the same function aliased) with `validate-app-labels` will be nice.